README.md

```diff
@@ -6,103 +6,255 @@ This firmware is in no way associated with The Button Corporation Ltd.
 * OTA update of the SPWF01SA Wi-Fi module
-* Custom led flashing by [POST request](https://github.com/mafredri/openbttn/commit/d2a22cb6291fbe04f809ac2fbe771ca0c1953c66)
-* Password authentication for bt.tn configuration
+* Password authentication for bttn configuration
 
-## Supported models
+## Hardware
 
-The following bt.tn models have been tested and confirmed working:
+#### Supported models
+
+The following bttn models have been tested and confirmed working:
 
 * bttn Wi-Fi
 
-### Untested models
+#### Untested models
 
 * bttn Mobile Data
 * bttn Mini Wi-Fi
 * bttn Mini Mobile Data
 
 ## Gettings started
 
-### First steps
+### Overview
+
+1. Download the latest release or build the project yourself
+2. (Optionally) backup the existing firmware
+3. Install new firmware
+4. Boot bttn in recovery mode to input your Wi-Fi settings
+5. Reboot your bttn
+5. Live configure the button press URLs using Web-UI
+6. Enjoy your OpenBttn!
 
-1. Download the latest [release](https://github.com/mafredri/openbttn/releases) or build the project yourself
-2. Install the firmware
-    * Optionally, backup the original firmware
-3. Configure bttn in recovery mode
-    1. Boot bttn while holding the button for 4 seconds to enter recovery mode
-    2. Connect to the 'OpenBttn' Wi-Fi network
-    3. Navigate to `http://192.168.1.1/`
-    4. Configure your Wi-Fi settings
-4. Configure the button press URLs
-    1. Navigate to `http://[bttn_IP_on_your_network]/`
-5. Enjoy your OpenBttn!
+### Get the firmware
 
-### Building
+You may compile the firmware yourself or use an pre-compiled firmware image.
+
+#### Download a pre-compiled firmware
+
+Grab the newest [release](https://github.com/mafredri/openbttn/releases) and unzip it.
+
+#### Build your own firmware from source
 
 Install the [GNU ARM Embedded Toolchain](https://launchpad.net/gcc-arm-embedded/+download):
 
 ```shell
 brew cask install gcc-arm-embedded
 ```
 
-Checkout and build the project:
+Check out and build the project:
 
 ```shell
 git clone https://github.com/mafredri/openbttn && cd openbttn
 git submodule update --init
-make  # builds src/
+make  # builds src/main.elf
+cd src && make bin # builds src/main.bin
+```
+
+### Firmware flashing methods
+
+This firmware can be installed in two ways, either via DFU mode or via the JTAG interface on the bttn board.
+
+Only DFU mode is covered here.
+
+### Boot bttn in DFU mode
+
+1. Open the bttn by removing the three T10 torx screws ![torx](resources/torx.png "Torx head") at the bottom.
+2. Connect the [Boot pins](./resources/bttn-boot-pins.png) on the board (note: only the Boot pins, not the FWSel ones). An easy way to do this is with [a paper clip and a bit of electrical tape](./resources/bttn-boot-pins-shorting.png) twice-folded so as to insulate the paper clip so that it does not touch the FWSel pins or any other contacts on the board.
```

Comment:
Here `./resources/bttn-boot-pins-shorting.png` should be `./resources/bttn-boot-pins-shorting.jpg`.